pkg/lib/config/authenticator.go

```diff
@@ -32,12 +32,20 @@ type AuthenticatorPasswordConfig struct {
 	Policy *PasswordPolicyConfig `json:"policy,omitempty"`
 }
 
+func (c *AuthenticatorPasswordConfig) SetDefaults() {
+	if c.Policy != nil {
+		if c.Policy.MinLength == nil {
+			c.Policy.MinLength = newInt(1)
+		}
+	}
```

Comment:
@kiootic Do we still have the populate-struct logic when we parse the config? 

I think we should also set default value if Policy is nil. In that case, initialize an empty Policy first.

Reply:
The populate default logic process the object graph in post-order traversal (i.e. populate recursively for the fields, then running the custom `SetDefaults` function), so there's no problem.

Reply:
So the SetDefaults method should be implemented on PasswordPolicyConfig?

Reply:
Yes, I suppose so.